node.gyp

```diff
@@ -220,6 +221,10 @@
               'defines': [ 'NODE_HAVE_SMALL_ICU=1' ],
           }]],
         }],
+        [ 'node_enable_v8_vtunejit=="true" and (target_arch=="x64" or target_arch=="ia32")', {
+          'defines': [ 'ENABLE_VTUNE_PROFILING' ],
```

Comment:
Can you call the define `NODE_ENABLE_VTUNE_PROFILING`?  Can you wrap the line above so it doesn't exceed 80 columns?